src/hooks/useTitle.ts

```diff
@@ -143,7 +142,6 @@
 // }
 
 // const routeMessages: Record<string, MessageDescriptor> = {
-//   [CONNECT_ROUTE]: MSG.connect,
```

Comment:
Remove the message descriptor as well: 

https://github.com/JoinColony/colonyCDapp/blob/2263ff3dc93d714c5d3a04f716fcdcb2679405c8/src/hooks/useTitle.ts#L38-L41